cmd/root.go

```diff
@@ -52,6 +54,16 @@ func getRootCmd(ks meta.IKubescape) *cobra.Command {
 		Example: ksExamples,
 	}
 
+	if strings.HasPrefix(filepath.Base(os.Args[0]), "kubectl-") {
```

Comment:
Let's find a better solution :)

Reply:
This is the official way: https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/

Reply:
I will move some logic into init and use `Sprintf` to also modify examples...

Reply:
@dwertent I have refactored a bit and modified all examples.